np7.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+def swap_last_first():
+    """
+    Function that create a 4x4 array with random values,
+    then create a new array from the said array swapping first and last rows
+    :return: Matrix
+    """
+    matrix = np.arange(16).reshape(-1, 4)
+    print(f"before:\n {matrix}")
+    new_matrix = np.vstack([matrix[matrix.shape[0]-1], matrix[1:3], matrix[0]])
+    print(f"after:\n {new_matrix}")
```

Comment:
Seperate IO from logic - don’t use print in functions unless there’s a good reason